### ISSUE TITLE
There is no usage of accessibility attributes in LiveEditor

### ISSUE THREAD

**Issue by yasin-herken** [FormidableLabs/react-live#374]
Can you add the accessibility attributes because Wave accessibility tool shows an error like missing form label
